Tests/Snapshotting+presentationLayer.swift

```diff
@@ -0,0 +1,41 @@
+// Created by Cal Stephens on 12/15/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+import SnapshotTesting
+import UIKit
+import XCTest
+
+extension Snapshotting where Value == UIView, Format == UIImage {
+  /// Captures an image of the input `UIView`'s `layer.presentation()`,
+  /// which reflects the current state of any in-flight animations.
+  static func imageOfPresentationLayer(
+    precision: Float = 1,
+    traits: UITraitCollection = .init())
+    -> Snapshotting<UIView, UIImage>
+  {
+    SimplySnapshotting
+      .image(precision: precision, scale: traits.displayScale)
+      .pullback { (view: UIView) -> UIImage in
```

Comment:
It's not entirely clear what pullback does from reading this—I assume it overrides the logic that goes from view to image but then does that augment the behavior from `.image` above? Might be helpful to explain exactly what's happening here

Reply:
haha you and me both 😆 

I was following the implementation of the framework's existing `UIView` -> `UIImage` snapshotting implementation, which is written like this:

```swift
return SimplySnapshotting.image(precision: precision, scale: traits.displayScale).asyncPullback { view in
  // ...
])
 ```